pkg/bitcoin/electrum/electrum_integration_test.go

```diff
@@ -295,6 +301,88 @@ func TestGetBlockHeader_Negative_Integration(t *testing.T) {
 	}
 }
 
+func TestGetTransactionMerkleProof_Integration(t *testing.T) {
+	transactionHash, err := bitcoin.NewHashFromString(
```

Comment:
Can we be consistent with other tests and define the transaction input and expected output in the test data package: "github.com/keep-network/keep-core/internal/testdata/bitcoin"?

Reply:
Moved the test data to `keep-core/internal/testdata/bitcoin/transaction.go`  in c647f1948545389e01ca9bb8b130e73db55aaf57.